aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/AuthComponentTest.java

```diff
@@ -553,6 +557,156 @@ public void confirmResetPassword() throws AuthException {
         verify(mobileClient).confirmForgotPassword(eq(NEW_PASSWORD), eq(CONFIRMATION_CODE), any());
     }
 
+    /**
+     * Tests that fetchUserAttributes method of the Auth wrapper of AWSMobileClient (AMC) calls
+     * AMC.getUserAttributes and get to fetch the user attributes.
```

Comment:
"and get to fetch the user attributes" ?? wording

Reply:
Yeah it should be "*to obtain a list of user attributes."😅